css/style.css

```diff
@@ -342,3 +345,143 @@ hr {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media only screen and (min-width: 175px) and (max-width: 420px) and (orientation: portrait) {
```

Comment:
- [ ] Please 🙏🏽 correct the media query, the value for the min-width property should be in the range of 768px or wider. It is because there are no media less than 375px for practical purposes, it represents the minimum screen size of mobile 📱, in the other hand we mark a point with CSS media queries to apply a different style which means if the width of the device is greater than 768px CSS will use the desktop 💻 styles.